manifests/pip.pp

```diff
@@ -66,11 +69,13 @@
 #
 # Sergey Stankevich
 # Fotis Gimian
+# Daniel Quackenbush
 #
 define python::pip (
   $pkgname         = $name,
   $ensure          = present,
   $virtualenv      = 'system',
+  $pip_proivder    = 'pip',
```

Comment:
Can you please add a dadatype for this new parameter? An enum would be awesome, otherwise String works as well.